doc/source/features/strict-mode.rst

```diff
@@ -0,0 +1,263 @@
+.. _strict-mode:
+
+===========
+Strict Mode
+===========
+
+This mode is designed to avoid the use of
+experimental and unsupported features in |PXC|.
+It performs a number of validations at startup and during runtime.
+
+Depending on the actual mode you select,
+upon encountering a failed validation,
+the server will either throw an error
+(halting startup or denying the operation),
+or log a warning and continue running as normal.
+The following modes are available:
+
+* ``DISABLED``: Do not perform strict mode validations
+  and run as normal.
+
+* ``PERMISSIVE``: If a vaidation fails, log a warning and continue running
+  as normal.
+
+* ``ENFORCING``: If a validation fails during startup,
+  halt the server and throw an error.
+  If a validation fails during runtime,
+  deny the operation and throw an error.
+
+* ``MASTER``: The same as ``ENFORCING`` except that the validation of
+  :ref:`explicit table locking <explicit-table-locking>` is not performed.
+  This mode can be used with clusters
+  in which write operations are isolated to a single node.
+
+By default, *enforcing* strict mode is enabled,
+except if the node is acting as a standalone server
+or the node is bootstrapping, then strict mode is disabled.
+
+To select the mode,
+use the :variable:`pxc_strict_mode` variable in the configuration file
+or the ``--pxc-strict-mode`` option during ``mysqld`` startup.
+
+.. note::
+
+   Although it is recommended to start the server with the necessary mode,
+   you can dynamically change it during runtime.
+   For example, to enable permissive strict mode::
+
+      mysql> SET pxc_strict_mode=PERMISSIVE;
+
+Validations
+===========
+
+Strict mode validations are designed to ensure optimal operation
+for common cluster setups that do not require experimental features
+and do not rely on operations not supported by |PXC|.
+If you are planning to disable strict mode,
+you should be aware of the limitations and effects
+that this may have on data integrity.
+
+.. warning:: If an unsupported operation is performed on a node
+   with strict mode disabled (or set to *permissive*),
+   it will not be validated on nodes where it is replicated to,
+   even if the destination node has *enforcing* or *master* strict mode enabled. 
+
+This section describes the purpose and consequences of each validation.
+
+.. contents::
+   :local:
+
+Storage engine
+--------------
+
+|PXC| currently supports replication only for tables
+that use a transactional storage engine (XtraDB or InnoDB).
+When you enable strict mode,
```

Comment:
s/When you enable strict mode/When you set pxc_strict_mode = ENFORCING/g

enable is bit confusing as variable can have multiple values.